lib/poparser/po.rb

```diff
@@ -36,6 +36,24 @@ def add(entry)
     end
     alias_method :<<, :add
 
+    # delete entry from po file
+    #
+    # @example
+    #   
+    #   delete(entry)
+    #
+    # @param entry  [PoParser::Entry] entry to delete
+    # @return [None]
+    def delete(entry)
+      puts entry.class.name
```

Comment:
I'm not a fan of leaving random sdtouts in code. Is this on purpose? If yes, it should contain some context. If it was just for testing, it should be removed.

Reply:
This was left in accidentally, and will be removed.